README.md

```diff
@@ -20,37 +20,65 @@ npm install d3fc
 ```
 
-## Documentation
-
-The project is split into sub-packages, see the package-specific README.md files in the [packages folder](https://github.com/d3fc/d3fc/tree/master/packages) for API-level documentation and usage guidelines.
+Test your installation using the [simple chart example](https://github.com/d3fc/d3fc/tree/master/examples/simple-chart).
 
-Examples can be found on the [project website](http://d3fc.io/) (these same examples are also available offline - see examples/README.md).
+## Documentation
 
-There are also a number of other documents and articles that provide a bit more background regarding the design principles of D3FC:
+The following getting-started guides are available -
+
+- [Building a Chart](https://github.com/d3fc/d3fc/tree/master/docs/building-a-chart.md) - a walk through building a chart with D3FC and what makes it different from other libraries.
+- [The Decorate Pattern](https://github.com/d3fc/d3fc/tree/master/docs/decorate-pattern.md) - the pattern D3FC employs, resulting in charting components that are both simple and flexible.
+- [Transitions](https://github.com/d3fc/d3fc/tree/master/docs/transitions.md) - how to use D3 transitions with D3FC.
+
+For API documentation see the corresponding package documentation -
+
+* [d3fc-annotation](https://github.com/d3fc/d3fc/tree/master/packages/d3fc-annotation/README.md)
+* [d3fc-axis](https://github.com/d3fc/d3fc/tree/master/packages/d3fc-axis/README.md)
+* [d3fc-brush](https://github.com/d3fc/d3fc/tree/master/packages/d3fc-brush/README.md)
+* [d3fc-chart](https://github.com/d3fc/d3fc/tree/master/packages/d3fc-chart/README.md)
+* [d3fc-data-join](https://github.com/d3fc/d3fc/tree/master/packages/d3fc-data-join/README.md)
+* [d3fc-discontinuous-scale](https://github.com/d3fc/d3fc/tree/master/packages/d3fc-discontinuous-scale/README.md)
+* [d3fc-element](https://github.com/d3fc/d3fc/tree/master/packages/d3fc-element/README.md)
+* [d3fc-extent](https://github.com/d3fc/d3fc/tree/master/packages/d3fc-extent/README.md)
+* [d3fc-financial-feed](https://github.com/d3fc/d3fc/tree/master/packages/d3fc-financial-feed/README.md)
+* [d3fc-group](https://github.com/d3fc/d3fc/tree/master/packages/d3fc-group/README.md)
+* [d3fc-label-layout](https://github.com/d3fc/d3fc/tree/master/packages/d3fc-label-layout/README.md)
+* [d3fc-pointer](https://github.com/d3fc/d3fc/tree/master/packages/d3fc-pointer/README.md)
+* [d3fc-random-data](https://github.com/d3fc/d3fc/tree/master/packages/d3fc-random-data/README.md)
+* [d3fc-rebind](https://github.com/d3fc/d3fc/tree/master/packages/d3fc-rebind/README.md)
+* [d3fc-sample](https://github.com/d3fc/d3fc/tree/master/packages/d3fc-sample/README.md)
+* [d3fc-series](https://github.com/d3fc/d3fc/tree/master/packages/d3fc-series/README.md)
+* [d3fc-shape](https://github.com/d3fc/d3fc/tree/master/packages/d3fc-shape/README.md)
+* [d3fc-site](https://github.com/d3fc/d3fc/tree/master/packages/d3fc-site/README.md)
```

Comment:
This links to the now deleted site package.

Reply:
Good spot! Thanks, I'll get that fixed.